src/Netflix/Main.java

```diff
@@ -0,0 +1,237 @@
+package Netflix;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Main Class File
+ * @author: Brighton Zhang
+ */
+
+public class Main {
+    static BufferedReader keyboard;
+    public static void main (String[] args) throws IOException {
+
+        // Variable and object instantiation
+        String strUser;
+        String strChoice;
+        int intFilm;
+        int intSlot;
+        int intCounter;
+        boolean blnSuperHero, blnCrime , blnAdventure, blnComedy, blnSciFi, blnDrama, blnHorror;
+        boolean blnMenu = true;
+        boolean blnList = false;
+        Scanner sc = new Scanner(System.in);        
+        ArrayList<String> filmList = new ArrayList<String>();
```

Comment:
A lost opportunity here by using a simple array list of strings to manage the list film list.  There is a clear case to have a UserList class to manage the list of films selected by the user.  At minimum, this should be and array list of Film objects